.gitignore

```diff
@@ -0,0 +1,23 @@
+__pycache__
```

Comment:
Not sure where you copied this from but I'd recommend just blindly copying `Python.gitignore` from [here](https://github.com/github/gitignore/blob/master/Python.gitignore) which looks a little more comprehensive.

Reply:
thanks :)